internal/model/sql/postgres/schema/hellper.sql

```diff
@@ -19,6 +19,7 @@ CREATE TABLE public.incident (
 	severity_level int4 NULL,
 	channel_name text NULL,
 	updated_at timestamp NOT NULL DEFAULT now(),
+  snoozed_at timestamp NULL,
```

Comment:
```suggestion
  snoozed_at timestamptz NULL,
```

Reply:
I am in doubt if we put the timestamp or timestamptz. :thinking:

Reply:
I recommend using the same pattern:

```
	[...]
	start_ts timestamptz NULL,
	end_ts timestamptz NULL,
	identification_ts timestamptz NULL,
	[...]
```